assignment1.txt

```diff
@@ -0,0 +1,20 @@
+1. What is the significance of return 0; at the end of the main program? Is it  really needed?
+The main function needs to return a number because ANSI standards dictate that
+main() must be preceded by int. The function returns 0 becasue that indicates   nothing went wrong during the course of main. None of this is really neccesary  for the code to function, it's just a standard put in place.  
+2. What, if anything is wrong with the following code?
+int main() {
+	float x = 41.5;
+	printf("The value of x is %d", x);
+}
+In the printf statement, %d is being used to try and represent something that isnot an integer variable. %f should be used to represent x because it is a float 
+value. There is also not a return statement, which is neccesary since there is
```

Comment:
Yup! Since this code prints output, you'd need #include <stdio.h> at the very top! Nice job 👍